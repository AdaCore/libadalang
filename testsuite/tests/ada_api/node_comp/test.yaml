driver: ada-api
main: main.adb
input_sources: [pkg.ads]
