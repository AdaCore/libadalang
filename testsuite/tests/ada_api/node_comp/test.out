D1: <ObjectDecl ["I"] pkg.ads:5:7-5:19>
D2: <ObjectDecl ["I"] pkg.ads:10:4-10:26>
Resolved: <| ObjectDecl ["I"] pkg.ads:5:7-5:19 [pkg.ads:8:4] |>
Done.
